nilearn/plotting/displays.py

```diff
@@ -414,6 +414,11 @@ def _add_markers(self, marker_coords, marker_color, marker_size, **kwargs):
             if not isinstance(marker_color, _utils.compat._basestring) and \
                     len(marker_color) != 1:
                 marker_color = marker_color[relevant_coords]
+            
+            if isinstance(marker_size, np.ndarray) or \
+                isinstance(marker_size, list):
```

Comment:
you can break the line inside parentheses rather than using a `\`
or maybe `not isinstance(marker_size, numbers.Number)`? this way any sequence will go through